CHANGELOG for v1.x.x.md

```diff
@@ -2,9 +2,181 @@
 
 #### This changelog consists the bug & security fixes and new features being included in the releases listed below.
 
+## **v1.3.0 (16th of February 2021)** - *Release*
+
+* [feature] - PayPal Smart Button for quick and fast payment.
+
+* [feature] - Email Marketing-Newsletter to send the newsletter to your valuable customers.
+
+* [feature] - Control on Displaying of out of stock Product.
+
+* [feature] - Minimum Order Amount by which it will necessary to meet their order subtotal to make a successful purchase.
+
+* [feature] - Add Video of Your Product.
+
+* [feature] - Maintenance Mode, let search engines know that your site is currently down.
+
+* [feature] - Buy Now at Product Page, Admin can easily enable/disable the Buy Now Button for both themes.
+
+* [feature] - laravel Debugger Mode, This will help the developer/users to see all the packages they are using inside the Webkul folder.
+
+* [enhancement] - 98 percent of duplicated queries removed.
```

Comment:
@rahulshukla-webkul 
duplicate queries PR https://github.com/bagisto/bagisto/pull/4315 still open & planned to add in 1.3.1 release https://github.com/bagisto/bagisto/milestone/1